.gitignore

```diff
@@ -12,3 +12,5 @@ modules/[Aa][Hh][Kk]_[Tt]ests
 .settings
 .vscode
 sdk/tools/winesync/winesync.cfg
+/build
```

Comment:
Why?

Reply:
folder generated by cmake

Reply:
Not using our default build system.

Reply:
i run configure script and build trought cmaketools on vscode